lib/cli/src/project_types.js

```diff
@@ -195,13 +195,11 @@ export const supportedTemplates = [
 ];
 
 const notInstallableProjectTypes = [
-  projectTypes.UNDETECTED,
-  projectTypes.ALREADY_HAS_STORYBOOK,
-  projectTypes.UPDATE_PACKAGE_ORGANIZATIONS,
+  PROJECT_TYPES.UNDETECTED,
+  PROJECT_TYPES.ALREADY_HAS_STORYBOOK,
+  PROJECT_TYPES.UPDATE_PACKAGE_ORGANIZATIONS,
 ];
 
-export const installableProjectTypes = Object.values(projectTypes)
+export const installableProjectTypes = Object.values(PROJECT_TYPES)
   .filter((type) => !notInstallableProjectTypes.includes(type))
   .map((type) => type.toLowerCase());
-
-export default projectTypes;
```

Comment:
👍 I'm a big fan of removing `default` export when possible

Reply:
me too hehe